### ISSUE TITLE
Create proof of existence transaction

### ISSUE THREAD

**Issue by gjermundgaraba** [EmpowerPlastic/empowerchain#512]
Depends on #508

This task is to set up the proof of existence transaction using `MsgCreateProof` from empowerchain-ts-client. It needs to be signed and broadcasted using the connected wallet.

**Comment by ba1ciu** [comment]
Already done so closing!
